<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проект: КИНОКАССА</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stack">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageCinema">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="labelCinema">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Кинотеатры:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableCinema">
              <property name="columnCount">
               <number>4</number>
              </property>
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Название</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Адрес</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Телефон</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="0" column="2">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="selectCinema">
                  <property name="text">
                   <string>Выбрать</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="backMain">
                  <property name="text">
                   <string>Назад</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageFilm">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="labelFilm">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Фильмы:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cmbGenre"/>
              </item>
              <item>
               <widget class="QDateEdit" name="dateFilm">
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="tableFilm">
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Название</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Жанр</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Рейтинг</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Время</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Кинотеатр</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>О фильме</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Премьера</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Продюсер</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Страна</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Год производства</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Оригинальное название</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Постер</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QTextEdit" name="infoFilm">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="poster">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="backCinema">
                <property name="text">
                 <string>Назад</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="selectFilm">
                <property name="text">
                 <string>Выбрать</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageSession">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="labelSession">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Сеансы:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLineEdit" name="lineFilmInfo">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableSession">
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Зал</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Время</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Свободно</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="backFilm">
                <property name="text">
                 <string>Назад</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="selectSession">
                <property name="text">
                 <string>Выбрать</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
